multinet/user.py

```diff
@@ -137,3 +138,21 @@ def filtered_user(user: User) -> FilteredUser:
 def copy_user(user: User) -> User:
     """Create and return a new instance of User."""
     return from_dict(User, dataclasses.asdict(user))
+
+
+def search_user(query: str) -> Cursor:
+    """Search for users given a partial string."""
+
+    coll = user_collection()
+    aql = read_only_db("_system").aql
+
+    query = f"""
+        FOR doc in {coll.name}
+          FILTER CONTAINS(LOWER(doc.name), LOWER('{query}'))
+            OR CONTAINS(LOWER(doc.email), LOWER('{query}'))
```

Comment:
Can we sanitize these parameters? For example, the following query string could be a problem, `'); <query to get user sessions>; --`

Reply:
Looks like [this](https://github.com/ArangoDB-Community/pyArango#queries--aql) is how you do it

Reply:
Good point, we definitely should sanitize this input (and all input to AQL queries for that matter). It does seem like using bind vars is preferred, instead of injecting the values with f-strings. I can try that out.

Reply:
Good catch, @JackWilb.

I don't think we need to sanitize *all* AQL input, since that's why we have the readonly user. But any time we're constructing an AQL query expression based on user input, we should definitely do it via bind vars or whatever the recommended secure method is.

Reply:
The other thought I had was, we could also just replace this AQL search with a brute force linear search as well. But since this is already here and we have a path to a secure usage of it, we can press forward.

Reply:
Agreed. Your point about the readonly user definitely helps us in cases where an attacker would try to delete/modify data, but not in the cases where an attacker would want to read sensitive information (such as session ids, other collections, etc.). Sanitizing all queries where there are user inputs does solve that problem.

Reply:
I've updated user search to use `bind_vars` (f116cd2), and created an issue to complete doing this throughout the codebase (#440).